browser/pages/confirm/controller.js

```diff
@@ -35,14 +35,10 @@ class ConfirmController {
     // IF the JDK is not Configured then you can't install devstudio
     $scope.$watch('checkboxModel.cdk.selectedOption', function watchCdkSelectionChange(nVal) {
       if(nVal=='install') {
-        if($scope.checkboxModel.vagrant.selectedOption == 'detected'
-          && !$scope.checkboxModel.vagrant.hasOption('detected')) {
-          $scope.checkboxModel.vagrant.selectedOption = 'install';
-        }
-        if($scope.checkboxModel.virtualbox.selectedOption == 'detected'
-          && !$scope.checkboxModel.virtualbox.hasOption('detected')) {
-          $scope.checkboxModel.virtualbox.selectedOption = 'install';
-        }
+        // if($scope.checkboxModel.virtualbox.selectedOption == 'detected'
+        //   && !$scope.checkboxModel.virtualbox.hasOption('detected')) {
+        //   $scope.checkboxModel.virtualbox.selectedOption = 'install';
+        // }
```

Comment:
CDK 2 never was dependent on virtualbox at installation time, all vagrant operations like registering VM images and installing vagrant plug-ins were working fine without virtualbox present. But to run CDK 2 one would need virtualbox or hyper-v on windows.

Here I just removed automatic selection of virtualbox if CDK 3 is selected for installation. That would let to install CDK 3 and manually decide (until there is no detection) what one is going to use virtualbox or Hyper-V as virtualization